test/rpc-integration/test-logTiming/test.sh

```diff
@@ -28,5 +28,5 @@ done
 echo "Running a request which gets stuck, with logTiming enabled"
 ${client} \
     execute $dir/state-c.execute ${client_args} -O log-timing=true | \
-    $JQ 'del(.result.logs[].time)' | \
+    jq 'del(.result.logs[].time)' | \
```

Comment:
since the script can now be invoked locally, I've removed `$JQ` and assume `jq` is on local path